gui/src/3D/controls/holographicSlate.ts

```diff
@@ -266,7 +267,11 @@ export class HolographicSlate extends ContentDisplay3D {
         const node = new Mesh("slate" + this.name, scene);
 
         this._backPlate = CreateBox("backPlate" + this.name, { size: 1 }, scene);
-        this._contentPlate = CreateBox("contentPlate" + this.name, { size: 1 }, scene);
+        const faceUV = new Array(6).fill(new Vector4(0,0,1,1));
+        if (scene.useRightHandedSystem) {
+            faceUV[0] = new Vector4(0,1,1,0);

```

Comment:
Nit - You can use .set or copyFromFloats instead of creating a new vector. Same things goes to the other vector assignment in the other file.

Reply:
Done!